app/controllers/api/v1/migrations/external_controller.rb

```diff
@@ -61,6 +61,28 @@ def create_user
           render_data status: :created
         end
 
+        # POST /api/v1/migrations/rooms.json
+        # Expects: { room: { :name, :friendly_id, :meeting_id, :last_session } }
+        # Returns: { data: Array[serializable objects] , errors: Array[String] }
+        # Does: Creates a room.
+        def create_room
+          user = User.find_by(email: room_params[:owner_email], provider: room_params[:owner_provider])
```

Comment:
I think it's best if we decrypt the params and assign it to a variable first `room_par = room_params`